src/Core/Domain/ValueObject/ConfigurableCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Domain\ValueObject;
+
+use InvalidArgumentException;
+use SprykerSdk\SdkContracts\Entity\CommandInterface;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\Entity\ConverterInterface;
+use SprykerSdk\SdkContracts\Entity\ExecutableCommandInterface;
+
+class ConfigurableCommand implements CommandInterface, ExecutableCommandInterface
+{
+    /**
+     * @var \SprykerSdk\SdkContracts\Entity\CommandInterface&\SprykerSdk\SdkContracts\Entity\ExecutableCommandInterface
+     */
+    protected CommandInterface $command;
+
+    /**
+     * @var bool|null
+     */
+    protected ?bool $stopOnError;
+
+    /**
+     * @var array|null
+     */
+    protected ?array $tags;
+
+    /**
+     * @param \SprykerSdk\SdkContracts\Entity\CommandInterface $command
+     * @param bool|null $stopOnError
+     * @param array<string>|null $tags
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function __construct(CommandInterface $command, ?bool $stopOnError = null, ?array $tags = null)
+    {
+        if (!($command instanceof ExecutableCommandInterface)) {
+            throw new InvalidArgumentException(sprintf('Command %s should be executable', get_class($command)));
+        }
+
+        $this->command = $command;
+        $this->stopOnError = $stopOnError;
+        $this->tags = $tags;
+    }
+
+    /**
+     * @return string
```

Comment:
Please, update method's docblock according to our conventions https://github.com/spryker-sdk/sdk/blob/master/docs/conventions.md#contracts

Reply:
it doesn't have any new spec here